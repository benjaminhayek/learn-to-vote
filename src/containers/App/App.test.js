import React from 'react';
import { App } from './index.js';
import { mapStateToProps, mapDispatchToProps } from './index.js';
import { shallow } from 'enzyme';

describe('APP', () => {
  let wrapper;
  let mockFetch;

  beforeEach(() => {
    mockFetch = jest.fn();
    wrapper = shallow(
      <App />
    )
  })

  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
})
