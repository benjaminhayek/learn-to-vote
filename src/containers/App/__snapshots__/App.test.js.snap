// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`APP should match the snapshot 1`] = `
<div
  className="App"
>
  <Header />
  <MemberContainer />
  <SenateContainer />
</div>
`;
